Parsers/Daily Motivation.js

```diff
@@ -0,0 +1,17 @@
+/*
+activation_example:!motivate
+regex:!motivate
+flags:gmi
+order:100
+stop_processing:false
+*/
+
+var quotes = [ //Add more number of quotes based on need
+    "The only way to do great work is to love what you do. – Steve Jobs",
+    "Success is not the key to happiness. Happiness is the key to success. – Albert Schweitzer",
+    "Believe you can and you're halfway there. – Theodore Roosevelt",
+    "The future belongs to those who believe in the beauty of their dreams. – Eleanor Roosevelt"
+];
```

Comment:
Add more than 4 quotes or call an API to get quotes.

Reply:
> Add more than 4 quotes or call an API to get quotes.

I have updated the code please review it.